cmd/updater/main.go

```diff
@@ -89,6 +90,7 @@ func gatherOptions() options {
 	flag.StringVar(&o.group, "test-group", "", "Only update named group if set")
 	flag.IntVar(&o.groupConcurrency, "group-concurrency", 0, "Manually define the number of groups to concurrently update if non-zero")
 	flag.IntVar(&o.buildConcurrency, "build-concurrency", 0, "Manually define the number of builds to concurrently read if non-zero")
+	flag.DurationVar(&o.wait, "wait", 0, "Ensure at least this much time has passed since the last loop (exit if zero).")
```

Comment:
Example flag argument (like '5m'?)

Reply:
Meh, we don't do that elsewhere, they can see how we're using it, `--help` says it is a duration, and its a well-known go pattern.

Can fix later if necessary.

/hold cancel

Reply:
Thought it might be, and good point. 👍